Runtime/Interaction/InteractableObject.cs

```diff
@@ -22,8 +22,8 @@ public class InteractableObject : XRGrabInteractable
         
         [SerializeField]
         private bool isUsable = true;
-
-        private Rigidbody rigidbody;
+        
+        private new Rigidbody rigidbody;
```

Comment:
Plox rename the field instead of hiding a mess under a carpet ;)

Reply:
Well, the original `rigidboy` is way less optimal and deprecated anyway :innocent: